modules/transformer.js

```diff
@@ -66,20 +66,22 @@ export default class Transformer {
         if (Array.isArray(items)) {
           items = items.filter((item) => item);
         }
-        // create the block
-        const block = WebImporter.Blocks.createBlock(document, {
-          name: WebImporter.Blocks.computeBlockName(type),
-          variants,
-          cells: items,
-        });
-        if (block) {
-          // add block to DOM
-          if (insertMode === 'append') {
-            main.append(block);
-          } else if (insertMode === 'prepend') {
-            main.prepend(block);
-          } else {
-            element.replaceWith(block);
+        if (items.length) {
```

Comment:
Don't add an empty block. Would there ever be a time when you'd want to create a block with now rows?

Reply:
Awesome!  I was going to suggest this for 'columns' at least.
I don't remember an empty block ever being useful.